startup/GafferSceneUI/gafferSceneExamples.py

```diff
@@ -0,0 +1,16 @@
+import Gaffer
+import GafferUI
+import GafferScene
+
+GafferUI.Examples.registerExample(
+	"Scene Processing/Per-location Transform Spreadsheet",
+	"$GAFFER_ROOT/resources/examples/perLocationTransformSpreadsheet.gfr",
```

Comment:
Can we move the files into the matching folders?

Reply:
202ed056e12259507a6864b11bcfae6ad716bd16